source/common/ssl/context_impl.cc

```diff
@@ -248,7 +248,8 @@ bool ContextImpl::verifyPeer(SSL* ssl) const {
   return verified;
 }
 
-bool ContextImpl::verifySubjectAltName(X509* cert, const std::string& subject_alt_name) {
+bool ContextImpl::verifySubjectAltName(X509* cert,
```

Comment:
I assume we have 100% coverage on this function now? Can you please get someone else who is familiar with TLS to give a +1 on the TLS part. I'm really not familiar with this stuff and don't feel comfortable approving the actual crypto aspect.